asciidoc-confluence-publisher-cli/src/main/java/org/sahli/asciidoc/confluence/publisher/cli/ArgumentsParser.java

```diff
@@ -34,6 +34,15 @@ String mandatoryArgument(String key, String[] args) {
                 .orElseThrow(() -> new IllegalArgumentException("mandatory argument '" + key + "' is missing"));
     }
 
+
+    boolean optionalBoolArgument(String key, String[] args) {
```

Comment:
Few things here:
- We prefer to avoid abbreviations in the code base, therefore I would prefer `optionalBooleanArgument` as the method name
- Instead of using `filter`, `findFirst` and `isPresent`, you could use `anyMatch` instead
- I am not so sure about the semantics with `startsWith`: are you expecting the parameter to have only a key, but no value? Then `equals` would be more appropriate. But I would rather go with a proper key-value parameter like `skipSslVerification=true` (maybe with a `skipSslVerification` shortcut), but also `skipSslVerification=false` should be supported to explicitly enable the SSL verification (e.g. based on a Maven property). 
- Please add tests to `ArgumentsParserTest` for all relevant cases